### DISCUSSION TITLE
Wiki Frage - Solar Passthrough is on and the Battery Drain Strategy is empty when full

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#587] [Q&A] [answered]
Hallo @schlimchen Bernhard,
kannst du bitte ein Auge auf die Wiki werfen?
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter-%E2%80%90-Solar-Passthrough#solar-passthrough-is-on-and-the-battery-drain-strategy-is-empty-when-full

Es ist mir nicht ganz klar was genau der Satz "battery discharge is disabled" bedeutet:
"In addition, Solar Power will be used to compensate for the currently used energy in cases where the **battery discharge is disabled**."

Wird vielleicht damit gemeint 
"In addition, Solar Power will be used to compensate for the currently used energy when **battery charge state reached the stop threshold limit** AND **battery charge state has not reached yet the start threshold limit**" ?
(Die Batterie war leer (stop threshold) und jetzt muss sie wieder geladen werden (bis aufs start threshold))

Das bedeutet also, der Unterschied zwischen 

- "Solar Passthrough is off" wenn "battery charge state reached the stop threshold limit":
   Der Wechselrichter bleibt AUS bis "battery charge state reached the start threshold limit"
   
- "Solar Passthrough is on and the Battery Drain Strategy is empty when full" wenn "battery charge state reached the stop threshold limit"
   Der Wechselrichter bleibt AN, liefert aber hoechstens die Leistung die der Victron liefert bis die Batterie wieder voll ist "battery charge state reached the start threshold limit"
   

Habe ich die Logik korrekt verstanden? Ist es verständlich wenn ich es so in die Wiki beschreiben wuerde? 🙈

**Comment by schlimmchen** [comment, marked as answer]
> Hallo @schlimchen Bernard,

*Bernhard :wink: 

> "In addition, Solar Power will be used to compensate for the currently used energy when battery charge state reached the stop threshold limit AND battery charge state has not reached yet the start threshold limit"

Naja, dieser Satz macht nicht so recht Sinn insofern als dass der letzte Halbsatz immer wahr ist wenn die stop threshold unterschritten wurde.

> battery discharge is disabled

Damit ist gemeint, dass der DPL entscheidet, die Batterie nicht zu entladen.

Ich konnte mir nicht helfen und hab die Wiki-Seite überarbeitet. Ich bin so (endlich) recht zufrieden damit. Was denkst du?

**Reply by Manos1966** [reply]
@schlimmchen Danke! 👍 

Ist es möglich auch den Text der README_onBattery.md dementsprechend anzupassen? (es ist der Text der ursprünglich in die Wiki  transferiert wurde).

Ich habe keine Erfahrung mit .md Seiten 🙈 
![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/50041639-5733-4202-b513-311bf24a1ad4)

**Reply by schlimmchen** [reply]
Siehe #588.
